clinical_refs/migrations/0001_initial.py

```
@@ -15,12 +15,17 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='ClinicalReference',
             fields=[
-                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('title', models.CharField(max_length=200)),
                 ('source', models.CharField(max_length=120)),
-                ('year', models.PositiveIntegerField()),
+                ('year', models.PositiveIntegerField(validators=[django.core.validators.MinValueValidator(1900)])),
```

Ruff (0.12.2)

Line too long (115 > 88)

(E501)